src/FormBuilder/resources/js/form-builder/src/blocks/fields/phone/index.tsx

```diff
@@ -0,0 +1,13 @@
+import settings from './settings';
+
+import {FieldBlock} from '@givewp/form-builder/types';
+
+/**
+ * @unreleased
+ */
+const phone: FieldBlock = {
+    name: 'givewp/phone',
```

Comment:
Let's actually change this to be specific to donor phone as it's not a generic phone block like ffm

```suggestion
    name: 'givewp/donor-phone',
```

Reply:
Resolved in 8401b98cda3228c20ed8c98152745717c78c2c57